src/app/component/search-builder/search-builder.component.scss

```diff
@@ -22,7 +22,27 @@ h3 {
 }
 
 .add-field-wrapper {
-  margin: 0 .5em;
+  margin: 0 .2em;
+}
+
+:host ::ng-deep .add-field,
+:host ::ng-deep .add-field .ui-dropdown-label,
+:host ::ng-deep .add-field .ui-dropdown-trigger {
+  -webkit-transition: all .2s ease-out !important;
```

Comment:
Is the "!important" really necessary? I would only use it if we see no other way.

Reply:
Why do you not use "bbmri-transition" on the elements? It does the same.

Reply:
I must overwrite the behaviour of the PimeNG framework (dropdown). That's why I must use !important AND cannont specify the behaviour by the CSS-class "bbmri-transition".